node/src/command.rs

```diff
@@ -38,54 +38,62 @@ use {
     std::{io::Write, net::SocketAddr},
 };
 
-fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn ChainSpec>, String> {
+fn load_spec(
+    id: &str,
+    para_id: Option<u32>,
+    container_chains: Option<Vec<String>>,
+    mock_container_chains: Option<Vec<ParaId>>,
+    invulnerables: Option<Vec<String>>,
+) -> std::result::Result<Box<dyn ChainSpec>, String> {
+    let para_id: ParaId = match para_id {
+        Some(x) => x.into(),
+        None => 1000.into(),
+    };
+    let container_chains = match container_chains {
+        Some(x) => x,
+        None => vec![],
+    };
+    let mock_container_chains = match mock_container_chains {
+        Some(x) => x,
+        None => vec![2000.into(), 2001.into()],
+    };
+    let invulnerables = match invulnerables {
+        Some(x) => x,
+        None => vec![
+            "Alice".to_string(),
+            "Bob".to_string(),
+            "Charlie".to_string(),
+            "Dave".to_string(),
+        ],
+    };
+
     Ok(match id {
         "dev" | "dancebox_dev" => Box::new(chain_spec::dancebox::development_config(
             para_id,
-            vec![],
-            vec![2000.into(), 2001.into()],
-            vec![
-                "Alice".to_string(),
-                "Bob".to_string(),
-                "Charlie".to_string(),
-                "Dave".to_string(),
-            ],
+            container_chains,
+            mock_container_chains,
+            invulnerables,
         )),
-        "" | "dancebox-local" => Box::new(chain_spec::dancebox::local_dancebox_config(
+        "" | "dancebox_local" => Box::new(chain_spec::dancebox::local_dancebox_config(
```

Comment:
Let's add both?

Reply:
They [use dashes](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/cli/src/command.rs#L99), but I changed it to underscore mostly so that I can use [`load_spec` on `BuildSpec`](https://github.com/moondance-labs/tanssi/pull/532/files#diff-8ce03a5a2b79f3021ff2d6322d13448db8cfd5fb9267517ab4b96d0d299ed45fR280) directly, because that's how they are in the chainspecs.
Nevertheless, we can still have the CLI with dashes and have a simple str replace from underscore

Reply:
@tmpolaczyk that's a good idea. We do need to change something, though, as right now it's very confusing that `dancebox-local` uses dash and `dancebox_dev` underscore